opentracing-examples/README.md

```diff
@@ -0,0 +1,14 @@
+# OpenTracing-Java examples
```

Comment:
We need a "Why does this exist" sort of thing... Could just be `## Module purpose` or whatever. My TL;DR is (1) something to test API changes against, (2) something to look at for common instrumentation patterns, (3) something to use for Tracer regression testing.

Reply:
... oh, and let's make a pointer in the main OT-Java README, too.

Reply:
should be pointer under "Development" section?